packages/drafts-realm/opportunity-form.gts

```diff
@@ -0,0 +1,602 @@
+
+    <style>
+      .subject {
+        margin: 0px;
+      }
+    </style>
+  </template>
+}
+
+class EditSecForStageField extends Component<typeof StageField> {
+  @tracked selectedCategory = {
+    name: this.args.model.category || 'None',
+  };
+
+  @tracked categoryPlaceholder = this.args.model.category || 'None';
+
+  @tracked categoryOptions = [
+    { name: 'None' },
+    { name: 'Qualification' },
+    { name: 'Needs Analysis' },
+    { name: 'Proposal' },
+    { name: 'Negotiation' },
+    { name: 'Closed Won' },
+    { name: 'Closed Lost' },
+  ] as Array<CategorySignature>;
+
+  @action updateCategory(type: { name: string }) {
+    this.selectedCategory = type;
+    this.args.model.category = type.name;
+  }
+
+  <template>
+    <CardContainer @displayBoundaries={{false}} class='card-container'>
+
+      <BoxelSelect
+        @searchEnabled={{true}}
+        @searchField='name'
+        @placeholder={{this.categoryPlaceholder}}
+        @selected={{this.selectedCategory}}
+        @onChange={{this.updateCategory}}
+        @options={{this.categoryOptions}}
+        class='select'
+        as |item|
+      >
+        <div>{{item.name}}</div>
+      </BoxelSelect>
+
+    </CardContainer>
+
+    <style>
+      .select {
+        padding: var(--boxel-sp-xs);
+        background-color: white;
+      }
+    </style>
+  </template>
+}
+
+class StageField extends FieldDef {
+  @field category = contains(StringField, {
+    description: `Selected Category`,
+  });
+
+  static edit = EditSecForStageField;
+  static embedded = EmbeddedSecForStageField;
+}
+
+/* Account */
+class EditSecForAccountField extends Component<typeof AccountField> {
+  @tracked selectedAccount = {
+    name: this.args.model.name || 'Select',
+  };
+
+  @tracked accountPlaceholder = this.args.model.name || 'Select';
+
+  get getAccountsNames() {
+    let allAccounts = this.args.model.accounts || [];
+    return allAccounts.map((o) => ({ name: o.accountName }));
+  }
+
+  @action updateAccount(type: { name: string }) {
+    this.selectedAccount = type;
+    this.args.model.name = type.name;
+  }
+
+  <template>
+    <CardContainer @displayBoundaries={{false}} class='card-container'>
+
+      <BoxelSelect
+        @placeholder={{this.accountPlaceholder}}
+        @selected={{this.selectedAccount}}
+        @onChange={{this.updateAccount}}
+        @options={{this.getAccountsNames}}
+        class='select'
+        as |item|
+      >
+        <div>{{item.name}}</div>
+      </BoxelSelect>
+
+    </CardContainer>
+
+    <style>
+      .select {
+        padding: var(--boxel-sp-xs);
+        background-color: white;
+      }
+    </style>
+  </template>
+}
+
+class EmbeddedSecForAccountField extends Component<typeof AccountField> {
+  <template>
+    <CardContainer @displayBoundaries={{false}} class='card-container'>
+      {{this.args.model.name}}
+    </CardContainer>
+
+    <style>
+      .card-container {
+        background: transparent;
```

Comment:
Do in another PR. 

- pass in @backgroundColor argument to CardContainer
- Update its usage ember-freestyle 
- use setCssVar to help you